kong/llm/drivers/cohere.lua

```diff
@@ -302,11 +267,7 @@ function _M.post_request(conf)
 end
 
 function _M.pre_request(conf, body)
-  -- check for user trying to bring own model
-  if body and body.model then
-    return false, "cannot use own model for this instance"
-  end
-
+  -- noop
```

Comment:
Why can this check be completely be removed in the cohere driver, but the anthropic driver still has a check (albeit only checking for matching names?)

Reply:
https://github.com/Kong/kong/pull/12903/files#r1575194343